packages/connect-examples/new-example/app/components/ui/Tooltip.tsx

```
@@ -0,0 +1,30 @@
+import * as React from "react"
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)